app/renderer/js/utils/logger-util.js

```diff
@@ -30,6 +30,20 @@ class Logger {
 			timestamp = this.getTimestamp;
 		}
 
+		fs.readFile(file, 'utf8', (err, data) => {
+			if (err) {
+				throw err;
+			}
+			const logs = data.split('\n');
+			// Keep top 300 files of log for each instance
+			const trimLines = logs.slice(Math.max(logs.length - 300, 0));
```

Comment:
I think we would want to keep the bottom 300 lines i.e. the latest logs.

Reply:
I think this does just that.. slice removes the top (total-300) lines and leaves the bottom 300.. does it not?

Reply:
Okay, it does that. I just went by the comments.

Reply:
Oh sorry, wrote wrongly in the comment.. will update that.